src/filesystem.cpp

```diff
@@ -11,7 +11,7 @@
 #if defined(ZEP_FEATURE_CPP_FILE_SYSTEM)
 
 #include <filesystem>
-namespace cpp_fs = std::filesystem;
+namespace cpp_fs = std::__fs::filesystem;
```

Comment:
Should this maybe be surrounded by an `#ifdef __APPLE__`, and leave the original in the `#else` branch? Doesn't seem likely to be portable...